src/compiler.jl

```diff
@@ -69,7 +70,10 @@ const InactiveFunctions = Set([Base.CoreLogging.logmsg_code,
                                Base.eps,
                                Base.nextfloat,
                                Base.prevfloat,
-                               Core.kwfunc
+                               Core.kwfunc,
+                               Random.rand,
```

Comment:
I am generally worried about these very generic opt-outs. I suspect if I let @femtomc think hard enough he could come up with a scenario where one would need to take the derivative through the state of a sampler, or something like that.

But let's cross that bridge later.